expo-maps/android/src/main/java/expo/modules/maps/MapsViewGestures.kt

```diff
@@ -0,0 +1,9 @@
+package expo.modules.maps
+
+interface MapsViewGestures {
```

Comment:
We named markers interface `Markers` so to keep it consistent we can name this `Gestures`, although it is very subjective thing and we can decide other way. Let me know which name is better in your opinion.

```suggestion
interface Gestures {
```

Reply:
I don't mind renaming this. I just wasn't aware of @ellwiii's PR at the time of writing this.